src/app/core/entity/model/entity.ts

```diff
@@ -168,6 +168,15 @@ export class Entity {
     return;
   }
 
+  /**
+   * whether this entity type can contain "personally identifiable information" (PII)
+   * and therefore should follow strict data protection requirements
+   * and offer a function to anonymize records.
+   *
+   * Default, we always take a "privacy-first" approach - assuming that the entity type can contain PII.
+   */
+  static hasPII: boolean = true;
```

Comment:
or should we invert this to `hasNoPII`?

I feel like hasPII is much more natural to read and understand. But as the default assumes hasPII = true ...

Reply:
yeah, considering the range of different entity types, including more technical one, that sounds smart :+1: 

In the setup UI we can still be default make this checked - or force the user to actively set it to either checked or unchecked when setting up a new entity type.